src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -167,6 +168,7 @@ public void testOrderRuleWithOneGroup() throws Exception {
         final String[] expected = {
             "4: " + getCheckMessage(MSG_LEX, "java.awt.Button.ABORT",
                 "java.io.File.createTempFile"),
+            "7: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "java.util.List"),
```

Comment:
thus Input is used only ones in Tests, that is good.
it is not easy to understand why violation here, till you read attentively config.
please do me a favor, and move config in sort form (example: https://github.com/checkstyle/checkstyle/pull/7251/files#diff-47f03c91c47ef1876b942d41928d6df7R1) to input and place "//warn" where violation is actually happening.

Reply:
Done. The config is placed at the end of input.